developer_manual/app/configuration.rst

```diff
@@ -4,7 +4,7 @@ Configuration
 
 .. sectionauthor:: Bernhard Posselt <dev@bernhard-posselt.com>
 
-The config allows the app to set global, app and user settings can be injected from the ServerContainer. All values are saved as strings and must be casted to the correct value.
+The config that allows the app to set global, app and user settings can be injected from the ServerContainer. All values are saved as strings and must be casted to the correct value.
 
```

Comment:
Yes, I agree. This is one of those silly English verbs. You can have "to be opened" "to be closed" but not "to be casted"! I will make that change also.